app/modules/users/parameters.py

```diff
@@ -14,6 +13,7 @@
 
 from app.extensions.api.parameters import PaginationParameters
 from app.extensions.api import abort
+from app.modules.submissions.models import Submission
```

Comment:
I think the idea is to avoid too many file scope imports of the models, Check with @bluemellophone but I think it causes circular reference problems. 
I've been including things like this in the functions

Reply:
Yeah, can also move it to the function instead :+1: